pom.xml

```diff
@@ -33,7 +33,7 @@
 	</modules>
 	
 	<properties>
-		<openMRSVersion>1.10.5</openMRSVersion>
+		<openMRSVersion>2.6.5</openMRSVersion>
```

Comment:
Curious about how you chose to build with this version

Reply:
Agreed... it looks like this will be the minimum Core version requires for O3 with this change, so we are saying that the latest O3 will depend on the latest point release of Core?  (Which may be okay, but we probably should explicitly decide).

Reply:
Ah, sorry @ibacher  posted before I saw your response.

Reply:
I've gone with 2.4.1, which seemed reasonable (every distro I'm aware of that's trying to take advantage of O3 is on at least 2.5.x; 2.4.0 unfortunately has a bug that makes tests fail IIRC), but I'm happy to lower it further if there's consensus on that.

Reply:
2.4.1 is fine with me.